apps/judicial-system/backend/src/app/modules/subpoena/dto/updateSubpoena.dto.ts

```diff
@@ -18,8 +18,8 @@ export class UpdateSubpoenaDto {
 
   @IsOptional()
   @IsString()
-  @ApiPropertyOptional({ type: String })
-  readonly serviceDate?: string
+  @ApiPropertyOptional({ type: Date })
+  readonly serviceDate?: Date
```

Comment:
Using date here catches errors earlier, an error would otherwise occur when writing to the database.